nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -412,3 +425,30 @@ def get_statistic_points(self, model: TModel) -> StatisticPointsContainer:
                                                       tensor_collector=stat_collector,
                                                       algorithm=MinMaxQuantization))
         return output
+
+    def _apply_model_type_pass(self, model_type: Optional[ModelType], quantizer_setup: SingleConfigQuantizerSetup,
+                               nncf_graph: NNCFGraph, device: TargetDevice) -> None:
+        """
+        Applies changes in-place into quantizer setup based on model_type and device parameters.
+
+        :param model_type: Model type parameter.
+        :param quantizer_setup: Quantizer setup which considered to update.
+        :param nncf_graph: Instance of NNCFGraph.
+        :param device: Target device.
+        :return: None
+        """
+        if model_type == ModelType.TRANSFORMER and device in [TargetDevice.ANY, TargetDevice.CPU,
```

Comment:
Since a whole device list is here, do we really need to keep this condition?
cc @l-bat